_config.yml

```diff
@@ -610,6 +610,7 @@ changyan:
   enable: false
   appid:
   appkey:
+  count: true
```

Comment:
A comment should be added otherwise it is hard to understand what "count" is.

Reply:
done